From d5c45b5fb11748acbfc9ee6c7dbfeb04408de53d Mon Sep 17 00:00:00 2001
From: naruse <naruse@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Sat, 1 Mar 2014 09:02:36 +0000
Subject: [PATCH] merge revision(s) 44370: [Backport #9576]

	hash.c: extra states

	* hash.c (rb_hash_reject): use words "extra states".


git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/branches/ruby_2_1@45227 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 hash.c    | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hash.c b/hash.c
index 66cd94b..9a5814a 100644
--- a/hash.c
+++ b/hash.c
@@ -1178,7 +1178,7 @@ struct shift_var {
 #endif
 	}
     }
-#if HASH_REJECT_COPY_MISC_ATTRIBUTES
+#if HASH_REJECT_COPY_EXTRA_STATES
     result = rb_hash_dup_empty(hash);
 #else
     result = rb_hash_new();
diff --git a/version.h b/version.h
index 8c8f900..59c80e3 100644
-- 
1.9.1

